Makefile

```diff
@@ -0,0 +1,149 @@
+# Makefile for Harbor-arm project
+#
+# Targets:
+#
+# all: 
+
+BUILDPATH=$(CURDIR)/src/github.com/goharbor/harbor
+
+# default goharbor/harbor make path
+HARBOR_MAKEFILE_PATH=$(BUILDPATH)/Makefile
+HARBOR_PHOTON_MAKEFILE_PATH=$(BUILDPATH)/make/photon/Makefile
+HARBOR_PHOTON_CHARTSERVER_COMPILE_PATH=$(BUILDPATH)/make/photon/chartserver/compile.sh
+HARBOR_PHOTON_REGISTRY_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/registry/Dockerfile.binary
+HARBOR_PHOTON_TRIVY-ADAPTER_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/trivy-adapter/Dockerfile.binary
+HARBOR_PHOTON_NOTARY_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/notary/binary.Dockerfile
+HARBOR_PHOTON_PORTAL_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/portal/Dockerfile
+HARBOR_PUSHIMAGE_PATH=$(BUILDPATH)/make/pushimage.sh
+
+# goharbor harbor github url
+HARBOR_SOURCE_URL=https://github.com/goharbor/harbor.git
+SRCPATH=src/github.com/goharbor/harbor
+HARBOR_TAG=v2.3.0
+
+# goharbor harbor build path
+MAKEPATH=$(BUILDPATH)/make
+
+#goharbor/harbor/make/Makefile
+MAKEFILEPATH_PHOTON=$(MAKEPATH)/photon
+
+# parameters
+BUILD_PG96=false
+BUILDTARGET=build
+DEVFLAG=true
+GOBUILDIMAGE=golang:1.15.12
+BUILDBIN=false
+NPM_REGISTRY=https://registry.npmjs.org
+# enable/disable chart repo supporting
+CHARTFLAG=false
+BUILDTARGET=build
+GEN_TLS=
+
+#versions
+REGISTRYVERSION=v2.7.1-patch-2819-2553-redis
+NOTARYVERSION=v0.6.1
+NOTARYMIGRATEVERSION=v4.11.0
+TRIVYVERSION=v0.17.2
+TRIVYADAPTERVERSION=v0.19.0
+
+# version of chartmuseum for pulling the source code
+CHARTMUSEUM_SRC_TAG=v0.13.1
+
+# version of chartmuseum
+CHARTMUSEUMVERSION=$(CHARTMUSEUM_SRC_TAG)-redis
+
+# version prepare
+# for docker image tag
+VERSIONTAG=dev-arm
+BASEIMAGETAG=dev-arm
+BUILD_BASE=true
+PUSHBASEIMAGE=false
+BASEIMAGENAMESPACE=goharbor
+# #input true/false only
+PULL_BASE_FROM_DOCKERHUB=true
+
+# for harbor package name
+PKGVERSIONTAG=dev-arm
+
+PREPARE_VERSION_NAME=versions
+
+# dependency binaries
```

Comment:
we should use BUILDBIN=true to call the goharbor makefile, and these deps can be removed.